code/modules/cargo/expressconsole.dm

```diff
@@ -120,7 +120,7 @@
 						CHECK_TICK
 					if(empty_turfs && empty_turfs.len)
 						var/LZ = empty_turfs[rand(empty_turfs.len-1)]
-						SSshuttle.points -= SO.pack.cost * 2
+						SSshuttle.points -= SO.pack.cost * 1.2

```

Comment:
I believe you'll also want to change this.

Reply:
i swear i have braincells